exonum_client/protobuf_loader.py

```diff
@@ -25,10 +26,12 @@ class ProtobufProviderInterface:
 
     def get_main_proto_sources(self) -> List[ProtoFile]:
         """Gets the Exonum core proto sources."""
+        logging.critical("Method `get_main_proto_sources` is not yet implemented.")
```

Comment:
I don't think that misuse of the interface class should be logged. Exception here is enough.